apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx

```diff
@@ -127,8 +128,48 @@ const DynamicConnectorForm = (props) => {
     );
 };
 
+const validate = (values) => {
+    const errors = {};
+    if (
+        parseInt(
```

Comment:
Would be better to extract this to a variable for more readability. Check other places as well.

Reply:
Resolved by 601c7fa